2.Views/WebAppPages/Settings/ProfilePage.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Todoly.Core.UIElements.Commons;
+using Todoly.Core.UIElements.Enums;
+using Todoly.Core.UIElements.Interfaces;
+using Todoly.Core.UIElements.Web;
+
+namespace Todoly.Views.WebAppPages;
+
+public class ProfilePage
+{
+    public ITypeable FullNameTextField => new TextField("", new Locator(LocatorType.Id, "FullNameInput"));
+    public ITypeable EmailTextField => new TextField("", new Locator(LocatorType.Id, "EmailInput"));
+    public IClickable OkButton => new Button("", new Locator(LocatorType.XPath, "//div[@class='ui-dialog-buttonset']//span[text()='Ok']"));
+    public IClickable CloseButton => new Button("", new Locator(LocatorType.XPath, "//div[@aria-labelledby='ui-dialog-title-settingsDialog'][contains(@style, 'block')]//ul[@id='settings_tabs']//span[text()='close']"));
+    public IClickable NonDisplayedCloseButton => new Button("", new Locator(LocatorType.XPath, "//div[@aria-labelledby='ui-dialog-title-settingsDialog'][contains(@style, 'none')]//ul[@id='settings_tabs']//span[text()='close']"));
```

Comment:
Isn't enough to check whether the CloseButton is not displayed in order to verify that the Settings modal has been closed? Why do you need the NonDisplayClosebutton element?

Reply:
I'm using it to check if the settings window is effectively closed when I update the profile